### PR TITLE
Dockerize project, create and push image via GitHub Actions

### PR DESCRIPTION
**Pull request by LuisNaldo7**
- add dockerfile
- add github workflow
- update readme

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: ci
```
Comment (TobiWo) [inline .github/workflows/ci.yml:1]: Please give the workflow and the filename a better name e.g. `docker`

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-alpine3.15
```
Comment (TobiWo) [inline docker/Dockerfile:1]: Please put Dockerfile to root directory

**Review by TobiWo — COMMENTED** [review]
Beside the two requested changes:
1. Please use semantic release keyword `ci` for commit messages otherwise a new feature release would be triggered
2. Please sign your commits
    * assuming you already created a gpg commit signer and setup github gpg signing the workflow for re-commiting signed commits could be:
      * `git rebase -i head~4`
      * your text editor will open
      * put the following line below each commit you want to sign and re-commit: `exec git commit --amend --no-edit -S`
      * close your editor
      * force push

File: README.md [hunk 3]
```diff
@@ -56,55 +56,81 @@ The following workflow is based on [miniconda/Anaconda](https://docs.conda.io/en
 
 1. Navigate to the root folder of the repository
 1. Execute
-  
-    ```bash
```
Comment (TobiWo) [inline README.md:60]: Please add the deleted space to all code blocks:
* more convenient with regards to only press tab while writing docs
* 3 spaces are very uncommon for indentation

File: .github/workflows/ci.yml (hunk 1 above)
Comment (LuisNaldo7) [reply .github/workflows/ci.yml]: renamed

**Pull request closed without merging**